src/Domain/Donation.php

```diff
@@ -316,16 +316,24 @@ private function __construct(string $amount, string $currencyCode, PaymentMethod
 
     public static function fromApiModel(DonationCreate $donationData, Campaign $campaign): Donation
     {
-
         $psp = $donationData->psp;
         assert($psp === 'stripe');
 
-        $donation = new self($donationData->donationAmount, $donationData->currencyCode, $donationData->paymentMethodType);
+        $donation = new self(
+            $donationData->donationAmount,
+            $donationData->currencyCode,
+            $donationData->paymentMethodType,
+        );
 
         $donation->setPsp($psp);
         $donation->setUuid(Uuid::uuid4());
         $donation->setCampaign($campaign); // Charity & match expectation determined implicitly from this
+
         $donation->setGiftAid($donationData->giftAid);
+        // `DonationCreate` doesn't support a distinct property yet & we only ask once about GA.
+        $donation->setTipGiftAid($donationData->giftAid);
```

Comment:
I see previously this was only set on `update`, not on donation create, and we're not calling update so much now. Is this fixing a separate bug that resulted from that oversight? 

Also I'm a bit confused about why it's a nullable property - are there clear semantics for the three values, or should we make it a non-null bool?